internal/functions/add-certificatethumbprintwifconfig.ps1

```diff
@@ -0,0 +1,40 @@
+function Add-WIFConfigAuthorityThumbprint 
+{
+    [CmdletBinding()]
+    param (
+        [Parameter(Mandatory = $true, Position = 1)]
+        [string]$CertificateThumbprint
+    )
+
+    try 
+    {
+        $wifConfigFile = Join-Path ([System.Environment]::ExpandEnvironmentVariables("%ServiceDrive%")) "\AOSService\webroot\wif.config"
+        
+        [xml]$wifXml = Get-Content $wifConfigFile
+
+        $authorities = $wifXml.SelectNodes('//system.identityModel//identityConfiguration//securityTokenHandlers//securityTokenHandlerConfiguration//issuerNameRegistry//authority[@name="https://fakeacs.accesscontrol.windows.net/"]');
+        
+        if($authorities.Count -lt 1)
+        {
+            throw "Only one authority should be found with the name https://fakeacs.accesscontrol.windows.net/"
```

Comment:
Use
Write-PSFMessage 
and 
Stop-PSFFunction -Stepsupward 1

Reply:
was under the empression that throw would trigger the catch block, and there the Write-PSFMessage is present.
But added